### ISSUE TITLE
Q: migrating event listeners from v2 to v3 || adding audio support?

### ISSUE THREAD

**Issue by d33tah** [asciinema/asciinema-player#159]
Hi! Thanks for releasing v3! I'm pretty excited about it because our hackerspace experimented with v2 for a low-bandwidth recording of our online talks, but found performance lacking. Sounds like the idea can come back to life in v3!

Consider the folowing v2 code:

```
			var audio = new Audio('./records/audio.ogg');
			var asciinemaPlayer = document.getElementById('player');
			let offset = 25*60 + 5;

			asciinemaPlayer.addEventListener('play', function(e) {
				audio.currentTime = this.currentTime + offset;
				audio.play();
			});
			asciinemaPlayer.addEventListener('pause', function(e) {
				audio.pause();
			});
```

How should I go about porting this to v3? So far I only tried renaming asciinemaPlayer to AsciinemaPlayer, but `addEventListener` is not exposed.

Full prototype can be found here:

https://github.com/hakierspejs/asciinema_player_with_audio/blob/169f058b6573dc9f5caf9fcb5b25706758f487db/data/index.html#L33

Thanks in advance for any pointers and have a great day! :)

**Comment by cipy** [comment]
so, does the v2 has audio play support? re: https://github.com/asciinema/asciinema-server/issues/63#issuecomment-1024532663 thanks 🥇

**Comment by sickill** [comment]
@d33tah Check out #173 , it brings back API for programmatic control. If it looks good enough then I'll release it shortly as new RC version.

Here's how you'd adjust your code to v3:

```javascript
var audio = new Audio('./records/audio.ogg');
var asciinemaPlayer = asciinemaPlayer.create('/demo.cast', document.getElementById('player'));
let offset = 25*60 + 5;

asciinemaPlayer.addEventListener('play', function(e) {
  audio.currentTime = this.getCurrentTime() + offset;
  audio.play();
});

asciinemaPlayer.addEventListener('pause', function(e) {
  audio.pause();
});
```

@cipy you can add audio support easily with a snippet similar to what @d33tah uses.

**Comment by sickill** [comment]
Merged and released #173 as 3.0.0-rc.3 :rocket:
